contribution/development_guide.md

```diff
@@ -176,19 +174,19 @@
      LSM - AppArmor
      ```
 
-     KubeArmor is designed for Kubernetes, which means that Kubernetes should be ready in your environment. If Kubernetes is not prepared yet, please refer to [Kubernetes installation guide](self-managed-k8s/README.md). KubeArmor also requires either Docker or Containerd since it internally uses its APIs. KubeArmor requires LSMs to operate properly; thus, please make sure that your environment supports LSMs \(at least, AppArmor\). Otherwise, KubeArmor will work as Audit-Mode with no container behavior restriction.
-
-      * Alternative Setup - Minikube
-
-        MiniKube does not support LSMs by default; thus, we provide the custom ISO image for testing KubeArmor on Minikube.  
-        Please follow the instructions in [Minikube installation guide](minikube/README.md).
+     KubeArmor is designed for Kubernetes, which means that Kubernetes should be ready in your environment. If Kubernetes is not prepared yet, please refer to [Kubernetes installation guide](self-managed-k8s/README.md).
```

Comment:
```suggestion
     KubeArmor is designed for Kubernetes environment. If Kubernetes is not setup yet, please refer to [Kubernetes installation guide](self-managed-k8s/README.md).
```

Reply:
We should mention about `host-only` dev setup too maybe? Should be simpler in case troubles setting up k8s.

Reply:
For that we have alternative setup